assets/prototype/domain/shared/dateRangeInput/momentDate.js

```diff
@@ -0,0 +1,29 @@
+import { Icon, Intent, Tag } from '@blueprintjs/core/lib/esm';
+import classNames from 'classnames';
+import moment from 'moment';
+import * as React from 'react';
+
+const FORMAT = 'dddd, LL';
+const FORMAT_TIME = 'dddd, LL LT';
```

Comment:
ya this should be globalized somewhere in a single file/variable else there's going to be an explosion of these constants all over the place

Reply:
small note here, credits go to the original example: https://github.com/palantir/blueprint/blob/develop/packages/docs-app/src/examples/datetime-examples/common/momentDate.tsx